onnxruntime/core/providers/cpu/ml/tree_ensemble_helper.cc

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/cpu/ml/tree_ensemble_helper.h"
+#include "core/common/common.h"
+#include "onnx/defs/tensor_proto_util.h"
+
+using namespace ::onnxruntime::common;
+using namespace std;
+namespace onnxruntime {
+namespace ml {
+
+#if !defined(ORT_MINIMAL_BUILD)
+
```

Comment:
Assuming this should be removed

Reply:
Thanks, I'm worried CI did not fail with a compilation issue. I also added a unit test to check that python frontend can load a model defined with the new specifications.